common/app/model/content.scala

```diff
@@ -632,7 +632,13 @@ object Video {
       "videoDuration" -> elements.videos.find(_.properties.isMain).map{ v => JsNumber(v.videos.duration)}.getOrElse(JsNull)) ++ AtomProperties(content.atoms)
 
 
-    val optionalOpengraphProperties = if(content.metadata.webUrl.startsWith("https://")) Map("og:video:secure_url" -> content.metadata.webUrl) else Nil
+    val optionalOpengraphProperties =
+      if(content.metadata.webUrl.startsWith("https://"))
```

Comment:
Is this an example where the else branch should never be hit?

Reply:
I'm not sure... let's ask @akash1810 ?

Reply:
Yes there are, sadly. We should to re-write them in Composer as there is no reason for them not to be `https`.